plop/collector.py

```diff
@@ -182,18 +183,19 @@ def main():
                 # interpreted as relative to this module.
                 global __package__
                 del __package__
-                exec f.read() in globals(), globals()
-    except SystemExit, e:
+                exec(f.read() in globals(), globals())
```

Comment:
That test is somewhat random since it relies on thread scheduling and timing. It checks that the numbers are within 25% of expectations, but sometimes it's further out than that. There's not much we can do about besides just loosening the thresholds.